### PR TITLE
Include instructions for PMC migrations commands

### PR DESCRIPTION
**Pull request by rowanmiller**
Update database error page to include instructions for using Package Manager Console commands to add/apply migrations.

### CODE REVIEW

**Comment by rowanmiller** [comment]
cc @divega

**Comment by divega** [comment]
:shipit:

File: src/Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore/Strings.resx [hunk 1]
```diff
@@ -144,20 +144,20 @@
   <data name="DatabaseErrorPage_ApplyMigrationsFailed" xml:space="preserve">
     <value>An error occurred applying migrations, try applying them from the command line</value>
   </data>
-  <data name="DatabaseErrorPage_HowToApplyFromCmd" xml:space="preserve">
-    <value>You can also apply migrations from the command line:</value>
+  <data name="DatabaseErrorPage_HowToApplyFromPMC" xml:space="preserve">
+    <value>In Visual Studio, you can use Package Manager Console apply pending migrations to the database:</value>
```
Comment (Eilon) [inline src/Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore/Strings.resx]: `...you can use *the* Package Manager Console *to* apply ...`

File: src/Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore/Strings.resx [hunk 2]
```diff
@@ -144,20 +144,20 @@
...
   </data>
   <data name="DatabaseErrorPage_MigrationsAppliedRefresh" xml:space="preserve">
     <value>Try refreshing the page</value>
   </data>
-  <data name="DatabaseErrorPage_NoDbOrMigrationsInfo" xml:space="preserve">
-    <value>From the command line, scaffold a new migration and apply it to the database:</value>
+  <data name="DatabaseErrorPage_NoDbOrMigrationsInfoPMC" xml:space="preserve">
+    <value>In Visual Studio, use Package Manager Console to scaffold a new migration and apply it to the database:</value>
```
Comment (Eilon) [inline src/Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore/Strings.resx]: Recommend inserting `the` before `PMC`

File: src/Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore/Strings.resx [hunk 3]
```diff
@@ -213,4 +213,19 @@
...
+    <value>PM&gt; Add-Migration [migration name]</value>
+  </data>
+  <data name="DatabaseErrorPage_ApplyMigrationsCommandPMC" xml:space="preserve">
+    <value>PM&gt; Update-Database</value>
+  </data>
+  <data name="DatabaseErrorPage_NoDbOrMigrationsInfoCLI" xml:space="preserve">
+    <value>Alternatively, you can scaffold a new migration and apply it from a command prompt at your project directory:</value>
+  </data>
+  <data name="DatabaseErrorPage_PendingChangesInfoCLI" xml:space="preserve">
+    <value>Alternatively, you can scaffold a new migration and apply it from a command prompt at your project directory:</value>
```
Comment (Eilon) [inline src/Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore/Strings.resx]: This string is identical to the one above it. Is that deliberate?

File: src/Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore/Strings.resx [hunk 4]
```diff
@@ -144,20 +144,20 @@
...
+  <data name="DatabaseErrorPage_NoDbOrMigrationsInfoPMC" xml:space="preserve">
+    <value>In Visual Studio, use Package Manager Console to scaffold a new migration and apply it to the database:</value>
   </data>
   <data name="DatabaseErrorPage_NoDbOrMigrationsTitle" xml:space="preserve">
     <value>Use migrations to create the database for {0}</value>
   </data>
-  <data name="DatabaseErrorPage_PendingChangesInfo" xml:space="preserve">
-    <value>Scaffold a new migration for these changes and apply them to the database from the command line:</value>
+  <data name="DatabaseErrorPage_PendingChangesInfoPMC" xml:space="preserve">
+    <value>In Visual Studio, use Package Manager Console to scaffold a new migration for these changes and apply them to the database:</value>
```
Comment (Eilon) [inline src/Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore/Strings.resx]: And here.

**Comment by Eilon** [comment]
🕐

File: src/Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore/Strings.resx (hunk 3 above)
Comment (rowanmiller) [inline src/Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore/Strings.resx]: It is, they are used in different places on the page, so I kept them separate to make it easier to adjust the strings in the future.

**Comment by rowanmiller** [comment]
Updated with feedback from @Eilon

File: src/Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore/Strings.resx (hunk 3 above)
Comment (Eilon) [inline src/Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore/Strings.resx]: Ah ok that makes sense; just wanted to make sure it wasn't an accident.

**Comment by rowanmiller** [comment]
Here are the three states of the page with the changes...

screen_nodatabase
screen_pendingmigrations
screen_pendingmodelchanges

File: test/Microsoft.AspNetCore.Diagnostics.FunctionalTests/DatabaseErrorPageSampleTest.cs [hunk 5]
```diff
@@ -32,7 +32,7 @@ public DatabaseErrorPageSampleTest(TestFixture<DatabaseErrorPageSample.Startup>
             // Assert
             var body = await response.Content.ReadAsStringAsync();
             Assert.Equal(HttpStatusCode.InternalServerError, response.StatusCode);
-            Assert.Contains("From the command line, scaffold a new migration and apply it to the database:", body);
+            Assert.Contains("In Visual Studio, use Package Manager Console to scaffold a new migration and apply it to the database:", body);
```
Comment (Eilon) [inline test/Microsoft.AspNetCore.Diagnostics.FunctionalTests/DatabaseErrorPageSampleTest.cs]: Does this test need to be updated?

**Comment by Eilon** [comment]
Screenshots look great.

File: test/Microsoft.AspNetCore.Diagnostics.FunctionalTests/DatabaseErrorPageSampleTest.cs (hunk 5 above)
Comment (rowanmiller) [inline test/Microsoft.AspNetCore.Diagnostics.FunctionalTests/DatabaseErrorPageSampleTest.cs]: Ah yes, running build.cmd at the moment and it just failed on that 😄

**Pull request merged**